setup.cfg

```diff
@@ -77,6 +77,7 @@ visualization =
 tests =
   hypothesis>=6.0.0  # Included in pytest-astropy. 6.0 is the first version to support disabling function-scoped fixture warning
   jplephem  # For some coordinates tests
+  opencv-python
```

Comment:
Does this provide binary wheels with opencv included?

Reply:
Its a large wheel download so I am going to say yes.